src/tareas.ts

```diff
@@ -0,0 +1,76 @@
+interface Tasks {
+  id?: string;
+  title: string;
+  description: string;
+  complete?: boolean;
+}
+
+class List {
+  tasks: Tasks[] = [];
+
+  addTask(id: string, title: string, description: string): void {
```

Comment:
quien te pasaria el id para crear la tarea? el id lo debes generar tu (BE), no puedes darle la responsabilidad al cliente (FE) de que haga eso